server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -960,6 +960,34 @@ public boolean doesConsumerExist(String uuid) {
         return result != 0;
     }
 
+    /**
+     * Given the Consumer UUIDs filter out & returns the non-existing & unique consumer UUIDs
+     *
+     * @param consumerUuids consumer UUIDs.
+     * @return set of consumer UUIDs that doesnt exists.
+     */
+    public Set<String> filterNonExistingConsumerIds(Iterable<String> consumerUuids) {
```

Comment:
The name change makes more sense if you were still actually filtering the input collection.

Reply:
Sure.